docs/Developers/Newcomers_Guide.rst

```diff
@@ -201,6 +201,9 @@ to prevent them.
 Step 5. Sending Your Changes
 ----------------------------
 
+Before starting to write your first commit, check out
+`our commit guideline documentation <http://coala.io/commit>`_.
```

Comment:
two is needed if you have the displayed text more than once iirc
for example the famous "here" needs double otherwise I get errors